test/components/envelope_test.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:turf/turf.dart';
+import 'package:test/test.dart';
+
+void main() {
+  final point = Feature<Point>(
+      geometry: Point(coordinates: Position.named(lat: 102.0, lng: 0.5)));
+
+  final line = Feature<LineString>(
+    geometry: LineString(coordinates: [
+      Position.named(lat: 102.0, lng: 0.5),
+      Position.named(lat: 103.0, lng: 1.5),
+      Position.named(lat: 104.0, lng: 2.5),
+    ])
+  );
+
+  final poly = Feature<Polygon>(
+      geometry: Polygon(coordinates: [
+    [
+      Position.named(lat: 101.0, lng: 0.0),
+      Position.named(lat: 101.0, lng: 1.0),
+      Position.named(lat: 100.0, lng: 1.0),
+      Position.named(lat: 100.0, lng: 0.0),
+      Position.named(lat: 101.0, lng: 0.0),
+    ],
+  ]));
```

Comment:
quick coding style suggestion: dart is using trailing comma for nicer formatting